clang/test/SemaSYCL/loop_unroll.cpp

```diff
@@ -32,11 +32,6 @@ void foo() {
   [[clang::loop_unroll(4)]]
   for (int i = 0; i < 10; ++i);
 
-  // expected-error@+2 {{incompatible loop unroll instructions: '#pragma unroll(4)' and '[[clang::loop_unroll(2)]]'}}
```

Comment:
I think Vladimir already made similar change here https://github.com/intel/llvm/pull/1890/commits/9947ba90421ce458181133376be6172128f8dfbd .

Reply:
Sure, I'll rebase the commit after pulldown is complete. Just wanted to make the changes clear.